src/lib/redux-cozy-client/CozyClient.js

```diff
@@ -54,14 +54,18 @@ export default class CozyClient {
     return { permissions, sharings }
   }
 
-  async createSharing (permissions, contactId, sharingType, description) {
-    return this.api.createSharing(permissions, contactId, sharingType, description)
+  async createSharing (permissions, contactIds, sharingType, description) {
+    return this.api.createSharing(permissions, contactIds, sharingType, description)
   }
 
   async revokeSharing (sharingId) {
     return this.api.revokeSharing(sharingId)
   }
 
+  revokeSharingForClient (sharingId, clientId) {
+    return this.api.revokeSharingForClient(sharingId, clientId)
```

Comment:
What is the difference between `revokeSharing` and `revokeSharingForClient`?

Reply:
The first deletes the sharing, the second only revokes it for one of many recipients of the sharing.

Reply:
Ohh, so the first one `deleteSharing` and the second `removeSharingRecipient`?
Maybe we could rename them this way, unless I am the only one who did not understand.